proto/Makefile

```diff
@@ -50,7 +50,7 @@ $(swagger_out): $(source_files) build/swagger
 
 # Update buf image for breaking change check.
 .PHONY: gen-buf-image
-gen-buf-image: check
+gen-buf-image:
```

Comment:
I need to break the buf image but it you run check before `gen-buf-image` it fails because there are breaking changes (ha)